src/presenters/business/business-email-change-presenter.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Formats data ready for presenting in the `/business-details` page
+ * @module businessEmailChangePresenter
+ */
+
+const businessEmailChangePresenter = (data, yar) => {
+  return {
+    notification: yar ? yar.flash('notification')[0] : null,
+    pageTitle: 'View and update your business details',
+    metaDescription: 'View and change the details for your business.',
+    businessEmail: data.changeBusinessEmail
```

Comment:
I don't think the data will be called changeBusinessEmail becuase it will be displaying the original business email

Reply:
It will be displaying newly entered value. that is the reason for the assignment here: 
businessEmail: data.changeBusinessEmail